src/components/paint-controls.js

```diff
@@ -18,6 +18,9 @@ AFRAME.registerComponent('paint-controls', {
     this.controller = null;
     this.modelLoaded = false;
 
+    this.onEnterVR = this.onEnterVR.bind(this);
```

Comment:
I think we need by default:

this.el.object3D.visible = false;

Reply:
Done